### PR TITLE
Fix-#22196-200 Response Code When Exception is Thrown During Bootstrap

### PR DESCRIPTION
**Pull request by abbasalis**
### Description

Now the application will throw a 500 error instead of 200 response when it terminates during the bootstrap process. 

So, now the error message will be displayed with the 500 response code.

### Manual testing scenarios

1. Do any code change to throw an error during the bootstrap process. 
2. Refresh the page.

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=22666) <br/>All committers have signed the CLA.

File: lib/internal/Magento/Framework/App/Bootstrap.php [hunk 1]
```diff
@@ -108,6 +108,13 @@ class Bootstrap
      */
     private $factory;
 
+    /**
+     * Application Object
+     *
+     * @var ObjectManagerFactory
```
Comment (phoenix128) [inline lib/internal/Magento/Framework/App/Bootstrap.php:114]: Should this be `AppInterface` ?
I would also recommend to change from `$app` to `$application`.

**Review by phoenix128 — CHANGES_REQUESTED** [review]
Hello @abbasalis ,
thank you for your contribution, would you please check my comments?

**Comment by sidolov** [comment]
@abbasalis , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for the collaboration!

**Pull request closed without merging**